src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,111 @@
-export const Pagination = () => {};
+// import { event } from "cypress/types/jquery";
+import { getNumbers } from '../../utils';
+import cn from 'classnames';
+
+type Props = {
+  start: (fun: number) => void;
+  total: string[];
+  perPage: string[];
+  currentPage: number;
+  onPageChange: (fun: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  start,
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const lastPage: number = Math.ceil(total.length / perPage.length);
+  const arrPages: number[] = getNumbers(1, lastPage);
+
+  function handlePageChange(page: number) {
+    if (page !== lastPage || page !== 1) {
+      onPageChange(page);
+      start(1 + perPage.length * (page - 1));
+    }
+  }
+
+  const next =
+    currentPage !== lastPage
+      ? () => handlePageChange(currentPage + 1)
+      : undefined;
```

Comment:
it is better to handle this condition inside the function rather then making such expression. Also avoid using undefined explictly